proposals/2024-09-01-Tags-Improvement.md

```diff
@@ -0,0 +1,154 @@
+# Tags Improvement
+
+
+## Metadata
+
+|Tag |Value |
+|---- | ---------------- |
+|Proposal |[2024-09-01-Tags-Improvement](https://github.com/OAI/OpenAPI-Specification/tree/main/proposals/{2024-09-01-Tags-Improvement.md})|
+|Authors|[Lorna Mitchell](https://github.com/lornajane)|
+|Review Manager | TBD |
+|Status |Proposal|
+|Implementations | |
+|Issues | [1367](https://github.com/OAI/OpenAPI-Specification/issues/1367), [{Iss}](https://github.com/OAI/OpenAPI-Specification/issues/{Iss}), |
```

Comment:
```suggestion
|Issues | [1367](https://github.com/OAI/OpenAPI-Specification/issues/1367) |
```

Reply:
Ah, no I wanted to link something else here. I'll fix it